ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -435,26 +435,17 @@ - (BOOL)removeAllForUserId:(NSString * __nonnull)userId
     return YES;
 }
 
-- (BOOL)removeAllForUserId:(NSString *)userId error:(ADAuthenticationError *__autoreleasing  _Nullable *)error
+- (BOOL)wipeAllItemsForUserId:(NSString *)userId error:(ADAuthenticationError *__autoreleasing  _Nullable *)error
 {
     AD_LOG_WARN(nil, @"Removing all items for user.");
     AD_LOG_WARN_PII(nil, @"userId <%@>", userId);
 
-    NSArray *items = [self allItems:nil];
-    if (!items)
-    {
-        return NO;
-    }
+    NSDictionary *query = @{ (id)kSecClass : (id)kSecClassGenericPassword,
+                             (id)kSecAttrAccount: [userId adBase64UrlEncode],
+                             (id)kSecAttrAccessGroup: _sharedGroup };
```

Comment:
Is sharedGroup always guaranteed to be non-nil?

Reply:
Yes, shared group is always non-nil after initialization.